src/main/java/duke/Duke.java

```
@@ -0,0 +1,220 @@
+                } catch (InvalidFile e){
+                    System.out.println("Invalid input in file");
+                }
+            }
+            s.close();
+        } catch (FileNotFoundException e) {
+            System.out.println("Cannot read from file");
+        }
+    }
+
+    private static void printHello() {
+        String logo = " ____        _\n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println(logo);
+        System.out.println(HORIZONTAL_LINE);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    private static void printBye() {
+        System.out.println(HORIZONTAL_LINE);
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    public static void task() {
+        Scanner in = new Scanner(System.in);
+        String inputCommand = in.nextLine();
+
+        while (true) {
+            try {
+                if (inputCommand.equals("list")) {
+                    showList();
+                } else if (inputCommand.contains("done")) {
+                    markAsDone(inputCommand);
+                } else if (inputCommand.contains("todo")) {
+                    addToDo(inputCommand);
+                } else if (inputCommand.contains("deadline")) {
+                    addDeadline(inputCommand);
+                } else if (inputCommand.contains("event")) {
+                    addEvent(inputCommand);
+                } else if (inputCommand.contains("delete")){
+                    deleteTask(inputCommand);
+                } else if(inputCommand.contains("bye")){
+                    try {
+                        saveData();
+                    } catch (IOException e){
+                    }
```

Avoid empty catch block, leave comment if you intend it to be.